consensus/src/context/difficulty.rs

```diff
@@ -232,26 +232,22 @@ impl DifficultyCache {
 
         difficulties.push(self.next_difficulty(current_hf));
 
-        let mut diff_info_popped = Vec::new();
-
         for (new_timestamp, hf) in blocks {
             timestamps.push_back(new_timestamp);
 
             let last_cum_diff = cumulative_difficulties.back().copied().unwrap_or(1);
             cumulative_difficulties.push_back(last_cum_diff + *difficulties.last().unwrap());
 
             if timestamps.len() > self.config.total_block_count() {
-                diff_info_popped.push((
-                    timestamps.pop_front().unwrap(),
-                    cumulative_difficulties.pop_front().unwrap(),
-                ));
+                timestamps.pop_front().unwrap();
+                cumulative_difficulties.pop_front().unwrap();
```

Comment:
This `diff_info_popped` was removed by clippy because it isn't being read/used after creation.

Reply:
Ah this must have been left over from something